Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -104,15 +112,6 @@ $ curl 10.100.28.138:8080
 <?xml version="1.0" encoding="UTF-8"?><ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><ID>anonymous</ID><DisplayName></DisplayName></Owner><Buckets></Buckets></ListAllMyBucketsResult>
 ```
 
-It is also possible to use the internally registered DNS name:
```

Comment:
Even I not quite sure, AFAIR the issues happen if the cluster has old OBCs prior to v1.11, since their configmap already has the service endpoint details. @BlaineEXE please confirm

Reply:
From the code

```
// Before v1.11, Rook created a Service and custom Endpoints that routed to external RGW
// endpoints. This causes problems if the external endpoint has TLS certificates that block
// connections to other endpoints. This also makes it impossible to create an external mode
// CephObjectStore that references another CephObjectStore's Service endpoint in the same
// Kubernetes cluster.

...

// For any legacy users that have an external mode CephObjectStore successfully using
// the Service/Endpoints and who have already created OBCs,we  leave the legacy
// Service/Endpoints in place. We need to update legacy services if the user edits the
// externalRgwEndpoint -- perhaps their RGW node changed IPs.
```